libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -134,11 +141,14 @@ void DocStringTagParser::handleCallable(
 )
 {
 	static set<string> const validEventTags = set<string>{"dev", "notice", "return", "param"};
+	static set<string> const validErrorTags = set<string>{"dev", "notice", "param"};
```

Comment:
What about this one? Since you changed https://github.com/ethereum/solidity/pull/10987#discussion_r596884364 I think the `return` tag should not be allowed here.

Reply:
Yes, we should fix it.

Reply:
I added it to https://github.com/ethereum/solidity/issues/11156